packages/config/src/bridges/index.test.ts

```diff
@@ -18,6 +21,47 @@ describe('bridges', () => {
       }
     })
   })
+  describe('references', () => {
+    describe('every contract has risk view code references', () => {
+      for (const bridge of bridges) {
+        try {
+          const discovery = new ProjectDiscovery(bridge.id.toString())
+
+          for (const [riskName, riskEntry] of Object.entries(
+            bridge.riskView ?? {},
+          )) {
+            const risk = riskEntry as ProjectRiskViewEntry
+            if (risk.sourceCodeReferences === undefined) continue
+
+            it(`${bridge.id.toString()} : ${riskName}`, () => {
+              for (const sourceCodeReference of risk.sourceCodeReferences ??
+                []) {
+                const referencedAddresses = getReferencedAddresses(
```

Comment:
This would make debugging easier
```suggestion
            describe(`${bridge.id.toString()} : ${riskName}`, () => {
              for (const sourceCodeReference of risk.sourceCodeReferences ??
                []) {
                it(sourceCodeReference.contractIdentifier, () => {
                  const referencedAddresses = getReferencedAddresses(
```